### PR TITLE
[AAE-865]Create card view text item PO

### PR DESCRIPTION
**Pull request by cristinaj**
**Please check if the PR fulfills these requirements**

> - [ ] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Comment by eromano** [comment]
> Can we replace these tests with unit tests or integration tests? Card View does not require multiple systems testing as for me and can be a super fast unit test suite instead. @eromano what do you think?

probably the unit test are already there

**Comment by cristinaj** [comment]
> Can we replace these tests with unit tests or integration tests? Card View does not require multiple systems testing as for me and can be a super fast unit test suite instead. @eromano what do you think?

What if we raise a different ticket for this? This PR is just adding a PO for card-view-text-item in order to use it also in task header page and refactored the old tests

**Comment by DenysVuika** [comment]
> > Can we replace these tests with unit tests or integration tests? Card View does not require multiple systems testing as for me and can be a super fast unit test suite instead. @eromano what do you think?
> 
> What if we raise a different ticket for this? This PR is just adding a PO for card-view-text-item in order to use it also in task header page and refactored the old tests

Yes, of course. That is not the request to changes for this PR

File: lib/testing/src/lib/core/pages/card-view/card-view-text-item.page.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+ */
+
+import { element, by, ElementFinder, Locator } from 'protractor';
+import { BrowserActions, BrowserVisibility } from '../../utils/public-api';
+
+export class CardTextItemPage {
+
+    rootElement: ElementFinder;
+    textField: Locator = by.css('input[data-automation-id*="card-textitem-editinput"]');
+    saveIcon: Locator = by.css('button[data-automation-id*="card-textitem-update"]');
```
Comment (adinapitul) [inline lib/testing/src/lib/core/pages/card-view/card-view-text-item.page.ts]: How about saveButton and clearButton ?

File: lib/testing/src/lib/core/pages/card-view/card-view-text-item.page.ts [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        await BrowserActions.click(toggleText);
+        await BrowserVisibility.waitUntilElementIsVisible(this.rootElement.element(this.textField));
+    }
+
+    async enterTextField(text: string): Promise<void> {
+        await BrowserVisibility.waitUntilElementIsVisible(this.rootElement.element(this.textField));
+        await BrowserActions.clearSendKeys(this.rootElement.element(this.textField), text);
+    }
+
+    async clickOnTextSaveIcon(): Promise<void> {
```
Comment (adinapitul) [inline lib/testing/src/lib/core/pages/card-view/card-view-text-item.page.ts]: Maybe call these clickOnSaveButton, clickOnClearButton

**Pull request merged**